### PR TITLE
Add whatsapp to post footer template

### PR DESCRIPTION
**Pull request by MuluhGodson**
## Fixes

Related to 
- [x] https://github.com/creativecommons/wp-theme-creativecommons.org/pull/20
by @brylie 

## Description
This adds the WhatsApp icon to the footer of posts as the share icons on the Figma mockups include Whatsapp and also deletes the initial `single.php` template file from this repository.

## Screenshots
image

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: inc/partials/entry/entry-footer.php [hunk 1]
```diff
@@ -15,9 +15,12 @@
 	<div class="column is-2">
 		<div class="share-entry margin-vertical-normal">
 			<h6>Share</h6>
-			<a href="<?php echo CC_Site::social_share( 'facebook', get_the_ID() ); ?>" class="share facebook"><i class="icon facebook colored"></i></a>
-			<a href="<?php echo CC_Site::social_share( 'twitter', get_the_ID() ); ?>" class="share twitter"><i class="icon twitter colored"></i></a>
-			<a href="<?php echo CC_Site::social_share( 'linkedin', get_the_ID() ); ?>" class="share linkedin"><i class="icon linkedin colored"></i></a>
+			<div class="is-flex is-3">
+				<a href="<?php echo CC_Site::social_share( 'facebook', get_the_ID() ); ?>" class="share column facebook"><i class="icon facebook colored"></i></a>
+				<a href="<?php echo CC_Site::social_share( 'twitter', get_the_ID() ); ?>" class="share column twitter"><i class="icon twitter colored"></i></a>
+				<a href="<?php echo CC_Site::social_share( 'whatsapp', get_the_ID() ); ?>" class="share column twitter"><i class="icon whatsapp colored"></i></a>
```
Comment (brylie) [inline inc/partials/entry/entry-footer.php:21]:
```suggestion
				<a href="<?php echo CC_Site::social_share( 'whatsapp', get_the_ID() ); ?>" class="share column whatsapp"><i class="icon whatsapp colored"></i></a>
```

**Review by brylie — CHANGES_REQUESTED** [review]

**Review by brylie — APPROVED** [review]

File: single.php [hunk 2]
```diff
@@ -1,37 +0,0 @@
-<?php
```
Comment (brylie) [inline single.php:1]: I just realized this change is out-of-scope for the pull request. There is a separate issue to deprecate post formats, which may allow us to remove this file. However, I suspect we will still want to define a `single.php` in the `creativecommons-base` project, so long as the `wp-theme-creativecommons.org` theme `single.php` takes precedence when enabled.

**Review by brylie — CHANGES_REQUESTED** [review]

File: single.php (hunk 2 above)
Comment (MuluhGodson) [reply single.php:1]: Okay this means I will have to port the footer elements over to the `single.php` file.  I think previously the `single.php` file was made up of different templates (author, header, footer)...I just maintained the templates. If they are to be deprecated then I will have to move them over to the `single.php` file.

I am referring to this folder: https://github.com/creativecommons/creativecommons-base/tree/main/inc/partials/entry

Comment (brylie) [reply single.php:1]: It is alright for the `creativecommons-base` and `wp-theme-creativecommons.org` to have different blog post layouts. The Figma mockups only pertain to the `wp-theme-creativecommons.org` in cases like this.

Comment (brylie) [reply single.php:1]: > I am referring to this folder: https://github.com/creativecommons/creativecommons-base/tree/main/inc/partials/entry

I have proposed that we deprecate these post formats https://github.com/creativecommons/project_creativecommons.org/issues/165

**Pull request closed without merging**